Makefile

```diff
@@ -1,13 +1,14 @@
 BIN     := speechly
 VERSION ?= latest
 SRC     = $(shell find cmd -type f -name '*.go')
+TAGS    = ""
 
 all: build test lint
 
 build: bin/speechly
 
 bin/speechly: $(shell git ls-files)
-	go build -ldflags="-X 'github.com/speechly/cli/cmd.version=$(VERSION)'" -o bin/speechly
+	go build -ldflags="-X 'github.com/speechly/cli/cmd.version=$(VERSION)'" -tags "$(TAGS)" -o bin/speechly
```

Comment:
As the linked library needs to be in a certain path, maybe the tag could be given if the library is available?